### ISSUE TITLE
'restart' command does not work in Firefox 45

### ISSUE THREAD

**Issue by geotrader** [vimperator/vimperator-labs#460]
i using 
・firefox 45 with chocolatey
・win10 64bit pro

I send "restart" command after firefox closed.
after it's not run, i waiting long time.

**Comment by timss** [comment]
According to #496, the same can be seen with the newest release 3.13.0 with Firefox 46.

**Comment by timss** [comment]
https://bugzilla.mozilla.org/show_bug.cgi?id=1269859

**Comment by kmorihiro** [comment]
How about commenting L988 out temporarily until Firefox fixes this problem?

https://github.com/vimperator/vimperator-labs/blob/master/common/content/liberator.js#L988

```js
services.get("obs").notifyObservers(null, "quit-application-granted", null);
```

**Comment by timss** [comment]
Sorry about the delay. I may be in over my head, and I haven't been able to set aside the necessary time for following up on this issue. Looking to do so before or during next weekend.

In the mean time, yes, there are things about this function that could be deleted to make the restart function work again, such as [L988](https://github.com/vimperator/vimperator-labs/blob/master/common/content/liberator.js#L988) or possibly [L995](https://github.com/vimperator/vimperator-labs/blob/master/common/content/liberator.js#L988).

**Comment by timss** [comment]
https://bugzilla.mozilla.org/show_bug.cgi?id=1269859#c4

Removing `quit-application-granted` entirely more so now seems like a possible solution, but I'd like to hear what the Mozilla maintainer has to say first.

**Comment by timss** [comment]
The underlying issue has finally been tracked down, and was indeed introduced in version 45. A fix has been released in 48.0b9 and later. 

For more information, please see the updates in the previous bug report as well as this one:

https://bugzilla.mozilla.org/show_bug.cgi?id=1284687

Until then, please restart your browser by starting it after closing using `:xall`, `:restart` etc., as you normally would if you're using a version affected.

**Comment by matreex** [comment]
@gkatsev ,the problem had been miraculously fixed on firefox v48.0. :100:
